### PR TITLE
(SERVER-2268) Update error messaging

### PR DESCRIPTION
**Pull request by amysahli**

### CODE REVIEW

File: lib/puppetserver/ca/create_action.rb [hunk 1]
```diff
@@ -169,7 +169,7 @@ def check_submit_result(result, certname)
           return true
         else
           @logger.err 'Error:'
-          @logger.err "    When certificate request submitted for #{certname}:"
+          @logger.err "    When certificate request submitted for #{certname} received:"
```
Comment (justinstoller) [inline lib/puppetserver/ca/create_action.rb]: This is missing single quotes around the certname.

**Comment by justinstoller** [comment]
Adding to the commit message why we're updating the error messaging would be helpful. We also hit this endpoint when revoking. You might want to wait until after we land https://github.com/puppetlabs/puppetserver-ca-cli/pull/17 and rebase.

File: lib/puppetserver/ca/action/sign.rb [hunk 2]
```diff
@@ -147,7 +147,7 @@ def sign_requested_certs(certnames,settings)
               success = false
             else
               @logger.err 'Error:'
-              @logger.err "    When download requested for #{result.inspect}"
+              @logger.err "    When attempting to sign certificate request '#{certname}' received:"
```
Comment (amysahli) [inline lib/puppetserver/ca/action/sign.rb]: My main reason for this PR was this typo...but while I was here I thought I would standardize all our sign error messaging.

File: lib/puppetserver/ca/action/create.rb [hunk 3]
```diff
@@ -203,7 +203,7 @@ def check_sign_result(result, certname)
             return true
           else
             @logger.err 'Error:'
-            @logger.err "    When signing request submitted for #{certname}:"
+            @logger.err "    When attempting to sign certificate request '#{certname}' received:"
             @logger.err "      code: #{result.code}"
```
Comment (Magisus) [inline lib/puppetserver/ca/action/create.rb]: Can we phrase all of them like this one? "When attempting to do X, received:"

File: lib/puppetserver/ca/action/sign.rb (hunk 2 above)
Comment (justinstoller) [reply lib/puppetserver/ca/action/sign.rb]: Gotcha, I was thinking (hence my comment about revoke) that the updates to puppetserver to support SANs affect all PUTs to the certificate_status endpoint similarly and that this PR re-evaluate all those error messages.

Breaking it up along user facing lines, sign, revoke, create, etc makes sense (vs implementation lines like http methods to an endpoint).

File: lib/puppetserver/ca/action/create.rb (hunk 3 above)
Comment (amysahli) [reply lib/puppetserver/ca/action/create.rb]: sure

**Comment by amysahli** [comment]
Closing this as the work will be contained in other upcoming tickets.

**Pull request closed without merging**